Dockerfile

```diff
@@ -4,6 +4,6 @@ RUN mkdir -p /go/src/github.com/bfirsh/whalebrew
 WORKDIR /go/src/github.com/bfirsh/whalebrew
 
 COPY . /go/src/github.com/bfirsh/whalebrew
-RUN go-wrapper download -t ./...
-RUN go-wrapper install
+RUN go-wrapper download -t ./... && \
+  go-wrapper install
```

Comment:
Perhaps it would be even better to change to a multi-stage build instead; after compiling, go is no longer needed, so the final stage could be `FROM scratch` (or eg `FROM alpine`